target/linux/ath79/image/generic.mk

```diff
@@ -888,6 +888,23 @@ define Device/yuncore_a770
 endef
 TARGET_DEVICES += yuncore_a770
 
+define Device/yuncore_cpe830
+  ATH_SOC := qca9531
+  DEVICE_VENDOR := YunCore
+  DEVICE_MODEL := CPE830
+  DEVICE_ALT0_VENDOR = YunCore
+  DEVICE_ALT0_MODEL = CPE830D
+  DEVICE_ALT1_VENDOR = KuWfi
+  DEVICE_ALT1_MODEL = CPE830
```

Comment:
It might be better to have KuWfi CPE830 as ALT0 since the list in make menuconfig will only show the first two entries (the others will require a look into details). However, this may change as the ALT syntax is still in PR stage.